Library/Homebrew/extend/ENV/std.rb

```diff
@@ -57,15 +57,20 @@ def setup_build_environment(formula = nil)
         # See https://github.com/Linuxbrew/linuxbrew/issues/841
         prepend_path "LD_LIBRARY_PATH", formula.opt_lib
         prepend_create_path "LD_LIBRARY_PATH", formula.prefix
-        prepend "LD_LIBRARY_PATH", formula.lib, File::PATH_SEPARATOR
+        prepend_path "LD_LIBRARY_PATH", formula.lib
       end
 
       # Set the search path for header files.
       prepend_path "CPATH", HOMEBREW_PREFIX/"include"
       # Set the dynamic linker and library search path.
-      append "LDFLAGS", "-Wl,--dynamic-linker=#{HOMEBREW_PREFIX}/lib/ld.so -Wl,-rpath,#{HOMEBREW_PREFIX}/lib"
```

Comment:
According to https://github.com/numpy/numpy/pull/8110/files the use of a comma after -rpath is a Mac-specific thing, but this is inside an `if OS.linux?` block.

I'm confused; it doesn't look like every stdenv-built executable had a bad RPATH as a result, but I find it hard to believe that we didn't notice this until now.